### DISCUSSION TITLE
HACKING.md seems to be out of date

### DISCUSSION THREAD

**Discussion by aochagavia** [rust-lang/regex#1057] [General]
I was going through [HACKING.md](https://github.com/rust-lang/regex/blob/master/HACKING.md) and realized there is some text related to benchmarks that is no longer accurate. Since the file was touched 4 years ago, there might be other things that need updating too.

I'm opening this here because none of the available issue categories seemed fitting.

**Comment by BurntSushi** [comment]
I created an issue #1058 for this.
